pkg/yang/types_builtin.go

```diff
@@ -932,38 +980,33 @@ func (r YangRange) Equal(q YangRange) bool {
 
 // Contains returns true if all possible values in s are also possible values
 // in r.  An empty range is assumed to be min..max.
+// TODO(wenbli): The current behaviour of this is confusing and unexplained by
+// comments: why does the presence of min/max always cause the range to be
```

Comment:
If there is `min...max` (i.e., minimum AND maximum) then that should mean everything is contained, right?

Reply:
the problem is 1..2 contains min..max, and that doesn't make sense to me:
```
		{
			r1: YangRange{R(1, 10)},
			r2: YangRange{R(useMin, useMax)},
			ok: true,
		},
```